image_builder.py

```diff
@@ -21,6 +21,7 @@ def build_and_push(dockerfile: str, tag: str, pipeline_id: str):
         buildargs={
             "pipeline": pipeline_id,
         },
+        platform="linux/amd64",
```

Comment:
Can you elaborate on the rationale for this change?

Reply:
I couldn't build on my M2 macbook without this.

That said it's not a great solution to add this arbitrarily everywhere. I reverted this, will add manually in the meanwhile.